packages/tracing/src/hubextensions.ts

```diff
@@ -237,6 +238,18 @@ function _autoloadDatabaseIntegrations(): void {
   }
 
   const packageToIntegrationMapping: Record<string, () => Integration> = {
+    apollo() {
+      const integration = dynamicRequire(module, './integrations/apollo') as {
+        Apollo: IntegrationClass<Integration>;
+      };
+      return new integration.Apollo();
+    },
+    graphql() {
+      const integration = dynamicRequire(module, './integrations/graphql') as {
+        GraphQL: IntegrationClass<Integration>;
+      };
+      return new integration.GraphQL();
+    },
```

Comment:
tbh I'd rather make these opt-in rather than auto-enabled.

Any possible way we can integration test this? We can do it in the PR moving forward as well.

Reply:
Can we make sure these are not auto-enabled?